export/app.yaml

```diff
@@ -0,0 +1,13 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+service: cloud-datastore-admin
+
+libraries:
+- name: webapp2
+  version: "latest"
+
+handlers:
+- url: /cloud-datastore-export
```

Comment:
Should this handle URLs with stuff after /cloud-datastore-export too? Like, it should 404 on /cloud-datastore-export/1?

Asking because I noticed such URL requests get sent here by dispatch.yaml but then don't seem to get handled here if I'm reading correctly.

Reply:
Hmm actually I think maybe dispatch.yaml just shouldn't include anything after /cloud-datastore-export, I don't see why it would need to. I'll change dispatch.yaml